tsextra.d.ts

```diff
@@ -0,0 +1,9 @@
+/// <reference path="./typings/typescript/typescript.d.ts" />
+
+// These are declarations that exist on the typescript object but aren't currently included in the typescript.d.ts. This
+// file, and its reference in index.ts, may be removed if/when these functions are included in typescript.d.ts.
+
+declare module 'typescript' {
+	export function createEmitHostFromProgram(program: Program): any;
+	export function emitFiles(resolver: EmitResolver, host: any, targetSourceFile?: SourceFile): EmitResult;
+}
```

Comment:
I’ve been informed these definitions exist in `typescript_internal.d.ts` so we can probably use that for the moment